cmd/server-mux.go

```diff
@@ -408,33 +408,26 @@ func initListeners(serverAddr string, tls *tls.Config) ([]*ListenerMux, error) {
 
 // ListenAndServe - serve HTTP requests with protocol multiplexing support
 // TLS is actived when certFile and keyFile parameters are not empty.
-func (m *ServerMux) ListenAndServe(certFile, keyFile string) (err error) {
-
-	tlsEnabled := certFile != "" && keyFile != ""
-
-	config := &tls.Config{
-		// Causes servers to use Go's default ciphersuite preferences,
-		// which are tuned to avoid attacks. Does nothing on clients.
-		PreferServerCipherSuites: true,
-		// Set minimum version to TLS 1.2
-		MinVersion: tls.VersionTLS12,
-	} // Always instantiate.
-
-	if tlsEnabled {
-		// Configure TLS in the server
-		if config.NextProtos == nil {
-			config.NextProtos = []string{"http/1.1", "h2"}
+func (m *ServerMux) ListenAndServe(tlsCert *tls.Certificate) (err error) {
```

Comment:
I think `Srv` is a duplicate meaning as this certificate is used only in server and there is no client TLS certificate we have now.